creator/slack.py

```diff
@@ -334,13 +334,21 @@ def make_study_message(studyObj):
             continue
 
         blocks = make_study_message(study)
-        if len(blocks) > 0:
+        len_blocks = len(blocks)
+        if len_blocks > 0:
+            SLACK_LIMIT = 50
             response = client.conversations_join(channel=channel_id)
 
+            text = f"There are new updates for study {study.name}."
+            if len_blocks > SLACK_LIMIT:
+                text += (
+                    f" There were more updates than Slack can handle in one"
+                    f" message. Only the last {SLACK_LIMIT} are presented."
+                )
             response = client.chat_postMessage(
                 channel=channel_id,
-                blocks=blocks,
-                text="There are new updates for study {study.name}.",
+                blocks=blocks[-SLACK_LIMIT:],
+                text=text,
```

Comment:
This is actually fallback text that won't be displayed if you pass blocks. Perhaps append a text block that says the same thing?

Reply:
See [Slack docs](https://api.slack.com/methods/chat.postMessage#text_usage)